### PR TITLE
add cagetory id to post  datarow

### PR DESCRIPTION
**Pull request by aronpc**
Fix category id in DataRowTableSeeder

### CODE REVIEW

**Comment by dimsanchuk** [comment]
Why type is "text"? If you are fixing this issue ([#835](https://github.com/the-control-group/voyager/issues/835)), I think type should be "select_dropdown".

File: publishable/database/seeds/DataRowsTableSeeder.php [hunk 1]
```diff
@@ -48,6 +48,21 @@ public function run()
             ])->save();
         }
 
+        $dataRow = $this->dataRow($postDataType, 'category_id');
+        if (!$dataRow->exists) {
+            $dataRow->fill([
+                'type'         => 'text',
```
Comment (marktopper) [inline publishable/database/seeds/DataRowsTableSeeder.php:54]: Please use the `select_dropdown` type, and add the `relationship` JSON option.

**Review by marktopper — CHANGES_REQUESTED** [review]

**Review by fletch3555 — COMMENTED** [review]
@marktopper, aren't we trying to phase-out the Pages/Posts/Categories stuff...? If so, I don't want these changes in there

Also, @aronpc, these aren't related to the original PR.  Please use separate git branches so you don't accidentally mess up an existing PR while working on something else.

**Comment by aronpc** [comment]
> Also, @aronpc, these aren't related to the original PR. Please use separate git branches so you don't accidentally mess up an existing PR while working on something else.

Thanks

**Pull request closed without merging**